internal/ip/ip.go

```diff
@@ -67,28 +67,28 @@ func IsPrivateSubnet(ipAddress net.IP) bool {
 }
 
 // GetIPAddressForRequest returns the real IP address of the request
-func GetIPAddressForRequest(r *http.Request) string {
+func AddressForRequest(r *http.Request) string {
 	for _, h := range []string{"X-Forwarded-For", "X-Real-Ip"} {
 		addresses := strings.Split(r.Header.Get(h), ",")
 		// march from right to left until we get a public address
 		// that will be the address right before our proxy.
 		for i := len(addresses) - 1; i >= 0; i-- {
-			ip := strings.TrimSpace(addresses[i])
+			addr := strings.TrimSpace(addresses[i])
```

Comment:
I improved this function in `master-stable` by adding tests. Perhaps the next step would be to move it out to a common package as right now there is another very similar copy of it in `internal-apis`.

Reply:
agreed